site/content/en/docs/formats/pascal_voc.md

```diff
@@ -212,6 +212,9 @@ datum import -o project -f voc -i ./VOC2012
 datum export -p project -f voc -- --tasks detection,classification
 ```
 
+- `--add-background ADD_BACKGROUND` - allow export dataset with/without
+adding background label (by default: `True`)
```

Comment:
Add a recommendation about how and when to use the parameter.

Reply:
I still don't understand how and when I, as a user, supposed to use this option. In the original VOC format there is no dedicated label map at all, so the description is confusing. Add where?